core/metadata.js

```diff
@@ -66,6 +67,7 @@ export type Metadata = {
   trashed?: true,
   incompatibilities?: *,
   ino?: ?number,
+  fileid?: ?string,
```

Comment:
I suppose we could expect this prop to disappear after some time using a recent enough Node.js version?

Reply:
I'm not sure. Nodejs will give us the fileId as a Big Int in a new field, it's probably safer to serialize it to a string in the `fileid` field.